### PR TITLE
Add style to user view profile-setting

### LINKED ISSUE #37: Profile Settings View
Give a design to the html layout, improvising the design.

THERE IS NO REFERENCE IMAGE

### PR DESCRIPTION
**Pull request by Zama15**
# Description

What did you implemented/Why did you implemented this:
Add setting-profile view
Remove email input on the first form due to duplicity

## Screenshots
VIew edited:
light mode
image
dark mode
image

### CODE REVIEW

**Review by MauricioNC — APPROVED** [review]

File: app/assets/stylesheets/pages/_user_config.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    }
+  }
+
+  &__danger button {
+    @include themify {
+      background-color: darken(theme('backgroundColor-tertiary'), 0%);
+      border: 1px solid theme('colorInverted');
+      color: theme('colorInverted');
+    }
+  }
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_user_config.scss]: Isn't there a way to just add a single themify the code repeats a lot

**Review by juperameza — CHANGES_REQUESTED** [review]
Also try to improve the UI 
Something feels off the inputs are the same - size as the buttons, and make the font of the label smaller.maybe that'll fix the issue with the inputs feeling small ans unclicklabe
Also add more font weight to the button

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,87 +1,92 @@
...
+
+    <div class="field">
+      <%= form.label :weight %>
+      <%= form.text_field :weight %>
+    </div>
+
+    <div class="field">
+      <%= form.label :height %>
+      <%= form.text_field :height %>
+    </div>
```
Comment (JuanequeX) [inline app/views/devise/registrations/edit.html.erb]: in each field add a `required: true` to have a end to end validation.

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -1,87 +1,92 @@
...
-  </div>
+    <h2>Extras</h2>
+    <div class="actions__warning">
+      <%= button_to "Sign Out", destroy_user_session_path, method: :delete %>
+    </div>
 
-  <div class="actions">
-    <%= f.submit "Update" %>
+    <div class="actions__danger">
+      <%= button_to "Delete my account", registration_path(resource_name), data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, method: :delete %>
```
Comment (JuanequeX) [inline app/views/devise/registrations/edit.html.erb]:
```suggestion
      <%= button_to "Delete my account", registration_path(resource_name), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" } %>
```

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (Zama15) [reply app/views/devise/registrations/edit.html.erb]: done :+1:

File: app/views/devise/registrations/edit.html.erb (hunk 3 above)
Comment (Zama15) [reply app/views/devise/registrations/edit.html.erb]: done :+1:

File: app/assets/stylesheets/pages/_user_config.scss (hunk 1 above)
Comment (Zama15) [reply app/assets/stylesheets/pages/_user_config.scss]: i did it, but i couldn't modify the themify in order to do not get error with the others files that use the themify, so i just added another mixin, maybe later in another pr i modify all the files and change the last mixin to the new one

Comment (juperameza) [reply app/assets/stylesheets/pages/_user_config.scss]: Gud

**Review by juperameza — APPROVED** [review]

**Review by angel23v — APPROVED** [review]

**Review by angel23v — COMMENTED** [review]
Change the theme!!!!

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**